server/models/Types.ts

```diff
@@ -0,0 +1,18 @@
+import { ObjectID } from "mongodb";
+
+export interface Types {
+  _id: ObjectID;
+  name: string;
+  description?: string;
+  date?: Date;
+  start_time?: Date;
+  end_time?: Date;
+  location?: string;
+  start_registration?: boolean;
+  end_registration?: boolean;
+  hours?: number;
+  image?: File;
```

Comment:
Can you make this a string? I know that seems confusing at first but the image will be stored at a certain url that we can reference, and the url is a string. When uploading the image, we'll have a form field that contains the File image itself, which will get uploaded to Contentful, which is what provides us the url field to use here.